### DISCUSSION TITLE
Pie and donut chart total

### DISCUSSION THREAD

**Discussion by hkphooey777** [EvotecIT/PSWriteHTML#173] [Q&A]
For pie and donut charts is there a way to add the total on the chart? For the donut in the middle would be awesome.

**Comment by PrzemyslawKlys** [comment]
It's possible in ApexCharts
- https://apexcharts.com/docs/chart-types/pie-donut/

This most likely requires a new cmdlet to be created and tested.

**Reply by hkphooey777** [reply]
ok thanks - your module is amazing btw

**Reply by PrzemyslawKlys** [reply]
Thank you :)

**Comment by PrzemyslawKlys** [comment]
Opened and issue: https://github.com/EvotecIT/PSWriteHTML/issues/175 - I guess unless I will have some time to work on it, nobody will touch it ;p

**Reply by hkphooey777** [reply]
I may take a stab at it on my own copy - will post here if I figure it out - thanks
